rush.json

```diff
@@ -386,6 +386,10 @@
     { "packageName": "@librocco/svg", "projectFolder": "pkg/svg" },
     { "packageName": "@librocco/ui-react", "projectFolder": "pkg/ui-react" },
     { "packageName": "@librocco/shared", "projectFolder": "pkg/shared" },
-    { "packageName": "js-search", "projectFolder": "pkg/js-search" }
+    { "packageName": "js-search", "projectFolder": "pkg/js-search" },
+    {
+      "packageName": "@librocco/web",
+      "projectFolder": "pkg/web"
+    }
```

Comment:
1. I feel that the db interfaces can be their own separate package, only assembled in the app. 
2. The `pkg/web` is deprecated and will be replaced with `apps/web-client`, so this should be moved out (my suggestion is 1. ☝️ )

I can help out with setting up of the package for db interfaces.